android/app/src/main/res/values/styles.xml

```diff
@@ -5,6 +5,7 @@
         <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/primaryColor</item>
         <item name="colorPrimaryDark">@color/primaryColorDark</item>
+        <item name="android:textColor">@color/black</item>
```

Comment:
This will also conflict with the idea of 'themes' where the color is not predetermined.